src/context.cc

```diff
@@ -2941,6 +2952,11 @@ error Context::SetTimeEntryStart(
 
     Poco::LocalDateTime local(Poco::Timestamp::fromEpochTime(te->Start()));
 
+    // Validate time input
+    if (local.year() < kMinimumAllowedYear || local.year() > kMinimumAllowedYear) {
```

Comment:
This condition is wrong in my case when I try to edit the Start Time from `17:30` to `17:20`. Therefore, it presents invalid Year error.

<img width="857" alt="Screen Shot 2019-04-02 at 10 47 49" src="https://user-images.githubusercontent.com/5878421/55375257-b46a3e00-5535-11e9-8d89-21b62d7634e5.png">

## Attempt:

<img width="901" alt="Screen Shot 2019-04-02 at 10 47 35" src="https://user-images.githubusercontent.com/5878421/55375326-ebd8ea80-5535-11e9-81cf-41981904057e.png">

The `local.year()` return `106` (in console), but `local._year` is `2019`

Reply:
For some reason, the `local.year()` returns wrong value event the TimeEntry's StartDate is correct 💭

Reply:
local.year actually returns the correct value. I'm not sure why console show other number. The issue was that in second check there should be `kMaximumAllowedYear`.